components/CardResult.js

```diff
@@ -24,20 +25,41 @@ export default class CardResult extends Component {
 
   state = {
     card: null,
+    allPrintings: null,
     rulings: [],
     faceIndex: 0,
     isOracleModalOpen: false,
     isRulingsModalOpen: false,
+    isPrintingsModalOpen: false,
+  };
+
+  selectCard = id => {
+    // get the card
+    const { allPrintings } = this.state;
+    const card = allPrintings.find(card => card.id === id);
+    this.setState({ card });
+    this.closePrintingsModal();
   };
 
   componentDidMount() {
     const { name } = this.props;
     fetch(
-      `https://api.scryfall.com/cards/named?exact=${encodeURIComponent(name)}`
+      `https://api.scryfall.com/cards/search?order=released&q=%21%E2%80%9C${encodeURIComponent(
+        name
+      )}%E2%80%9D+include%3Aextras&unique=prints`
     )
       .then(result => result.json())
-      .then(card => {
+      .then(result => {
+        const cards = result.data;
+        const card = cards[0];
         this.setState({ card });
+        if (cards.length > 1) {
+          const allPrintings = [];
+          cards.forEach(card => {
+            allPrintings.push(card);
+          });
+          this.setState({ allPrintings });
```

Comment:
Can we just do `this.setState({ allPrintings: cards })`?

Reply:
Or at least `cards.slice()` if we try to avoid using the result data for some reason?

Reply:
For some reason when I was writing this, it was the way to go but now that you mention it, it seems superfluous to not just do `this.setState({ allPrintings: cards })`